node/src/accountant/mod.rs

```diff
@@ -2608,25 +3299,50 @@ pub mod tests {
         ));
     }
 
+    #[test]
+    #[should_panic(
+        expected = "Recording services provided for 0x000000000000000000000000000000626f6f6761 \
+    but has hit fatal database error: RusqliteError(\"we cannot help ourself; this is baaad\")"
```

Comment:
"Ourself" isn't a word, because "our" is plural and "self" is singular. (At least, it _shouldn't_ be a word.)